src/mame/handheld/hh_sm510.cpp

```diff
@@ -1626,6 +1659,11 @@ void gnw_mmouse_state::ecircus(machine_config &config)
 	kb1013vk12_common(config, 1657, 1080); // R mask option ?
 }
 
+void gnw_mmouse_state::naltair(machine_config &config)
+{
+	kb1013vk12_common(config, 1655, 1080); // R mask option ?
```

Comment:
I see it's the same SVG screen as vfutbol, then you can use the vfutbol mcfg instead of copy pasting it for naltair.